xen.spec.in

```diff
@@ -570,9 +469,8 @@ find %{buildroot} -print | xargs ls -ld | sed -e 's|.*%{buildroot}||' > f1.list
 rm -rf %{buildroot}/usr/*-xen-elf
 
 # hypervisor symlinks
-rm -rf %{buildroot}/boot/xen-4.10.gz
+rm -rf %{buildroot}/boot/xen-4.0.gz
```

Comment:
This looks like a bug. Should be 4.12 (and soon 4.13).

Reply:
I put %{upstream_version} macro.

Reply:
`%{upstream_version}` contains full version, not only first two parts. Maybe better `%{hv_abi}` (but it's weird name to use like this)?

Reply:
IMHO, it's fine.